sled-agent/src/rack_setup/service.rs

```diff
@@ -1185,8 +1215,10 @@ impl ServiceInner {
             },
         };
         info!(self.log, "Writing Rack Network Configuration to bootstore");
+        rss_step.update(RssStep::NetworkConfigUpdate);
         bootstore.update_network_config(early_network_config.into()).await?;
 
+        rss_step.update(RssStep::SledInit);
```

Comment:
Probably not for this PR, but just idly thinking - could we include how many sleds we've initialized and how many we're still waiting on? (Same question applies to other steps too.)

Reply:
I think we could tell how many sleds we want to initialize, `sled_addresses.len()` perhaps?  But to get a running count as things change we would have to dive into what `ServicePlan` is doing... I think..

Reply:
That might be worth it? But seems like a thing for followup work. "Now that we have _this_ level of detail, I want _more_ detail" kind of thing.

Reply:
Yeah, I like the idea of building more into this, as we deem it worth the effort. 

I was also thinking about:
1) Logging every state change, so a record exists as we transition to each step.
2) Recording the time when we start a step, so the status could report how long we have been in the current step.
3) A way to get a "RSS report" after the fact that includes total time to RSS as well as time in each step.